pyproject.toml

```diff
@@ -1,11 +1,79 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools @ git+https://github.com/abravalheri/setuptools@support-pyproject",
```

Comment:
@henryiii, you can also try now `git+https://github.com/pypa/setuptools@experimental/support-pyproject`.

However this branch also adds automatic discovery for packages (which supports `src-layout` without config) and considers implicit namespaces by default. So the `experiments` folder might end-up being included in the wheel.

Reply:
The exclusion rules are documented here: https://setuptools--2894.org.readthedocs.build/en/2894/userguide/package_discovery.html#automatic-discovery